security/middleware.py

```diff
@@ -696,37 +829,58 @@ def process_request(self, request):
 
         # If the session has no start time or last activity time, set those
         # two values. We assume we have a brand new session.
-        if (self.START_TIME_KEY not in request.session or
-                    self.LAST_ACTIVITY_KEY not in request.session or
-                timezone.is_naive(request.session[self.START_TIME_KEY]) or
-                timezone.is_naive(request.session[self.LAST_ACTIVITY_KEY])):
-
-            logger.debug("New session %s started: %s" % (request.session.session_key, now))
+        if (
+            self.START_TIME_KEY not in request.session or
+            self.LAST_ACTIVITY_KEY not in request.session or
+            timezone.is_naive(request.session[self.START_TIME_KEY]) or
+            timezone.is_naive(request.session[self.LAST_ACTIVITY_KEY])
+        ):
+            logger.debug("New session %s started: %s" % (
+                request.session.session_key,
+                now,
+            ))
             request.session[self.START_TIME_KEY] = now
             request.session[self.LAST_ACTIVITY_KEY] = now
         else:
             start_time = request.session[self.START_TIME_KEY]
             last_activity_time = request.session[self.LAST_ACTIVITY_KEY]
-            logger.debug("Session %s started: %s" % (request.session.session_key, start_time))
-            logger.debug("Session %s last active: %s" % (request.session.session_key, last_activity_time))
+            logger.debug("Session %s started: %s" % (
+                request.session.session_key,
+                start_time,
+            ))
+            logger.debug("Session %s last active: %s" % (
+                request.session.session_key,
+                last_activity_time,
+            ))
 
             # Is this session older than SESSION_COOKIE_AGE?
             # We don't worry about microseconds.
-            SECONDS_PER_DAY = 86400
+            seconds_per_day = 86400
```

Comment:
This was all caps because it was a 'constant'. I thought this was common practice.